### ISSUE TITLE
Cannot resolve libcrypto.so.1.0.0 when compiling built-in examples of python helloworld

### ISSUE THREAD

**Issue by gracequeen** [oscarlab/graphene#879]
Hi. 

I followed the instructions to build and run your built-in test examples. The compiling and run work properly with the native Graphene on SGX 2.1.3(1). The process failed at 2.1.3.(2). When I typed "make SGX=1", I had the following error:

```
ln -sf ../../../../../Runtime/pal_loader
tar -xmzf benchmarks.tar.gz
[ python.manifest ]
RELDIR= && \
sed -e 's:\$(PAL):/home/xi/graphene/Runtime/pal_loader SGX:g' -e 's:\$(PWD):/home/xi/graphene/LibOS/shim/test/apps/python:g' -e 's:\$(BIN):python:g' -e 's:\$(SHIMPATH):'$RELDIR'../../../../../Runtime/libsysdb.so:g' -e 's:\$(LIBCDIR):'$RELDIR'../../../../../Runtime:g' -e 's:\$(PYTHONDIR):Python-2.7.9/build/:g' python.manifest.template > python.manifest
(grep -q '^#!' python.manifest && chmod +x python.manifest) || true
for f in `sed -e 's/#.*//g' python.manifest | grep -Po 'sgx.trusted_children.[^\\s=]+\\s*=\\s*file:\\K\\S+'`; do make ${f%.sig}.manifest.sgx; done
make[1]: Entering directory '/home/xi/graphene/LibOS/shim/test/apps/python'
make[1]: 'python.manifest' is up to date.
make[1]: Leaving directory '/home/xi/graphene/LibOS/shim/test/apps/python'
[ python.manifest.sgx ]
/home/xi/graphene/Pal/src/host/Linux-SGX/signer/pal-sgx-sign -libpal /home/xi/graphene/Pal/src/host/Linux-SGX/../../../../Runtime/libpal-Linux-SGX.so -key /home/xi/graphene/Pal/src/host/Linux-SGX/signer/enclave-key.pem -output python.manifest.sgx -manifest python.manifest
Attributes:
    size:      268435456
    threadnum: 4
    isvprodid: 0
    isvsvn:    0
    flags:     0600000000000000
    xfrms:     0700000000000000
    miscs:     00000000
Trusted files:
Traceback (most recent call last):
  File "/home/xi/graphene/Pal/src/host/Linux-SGX/signer/pal-sgx-sign", line 822, in <module>
    for key, val in get_trusted_files(manifest, args).items():
  File "/home/xi/graphene/Pal/src/host/Linux-SGX/signer/pal-sgx-sign", line 231, in get_trusted_files
    targets[key] = (val, resolve_uri(val))
  File "/home/xi/graphene/Pal/src/host/Linux-SGX/signer/pal-sgx-sign", line 202, in resolve_uri
    raise Exception('Cannot resolve ' + orig_uri + ' or the file does not exist.')
Exception: Cannot resolve file:/lib/x86_64-linux-gnu/libcrypto.so.1.0.0 or the file does not exist.
../../../../../Pal/src//host/Linux-SGX/Makefile.Test:40: recipe for target 'python.manifest.sgx' failed
make: *** [python.manifest.sgx] Error 1
```

Could you help with this? Thanks!

**Comment by dimakuv** [comment]
Do you have the file `/lib/x86_64-linux-gnu/libcrypto.so.1.0.0` on your computer? Doesn't look like it. You probably should install it (`sudo apt-get install libssl1.0.0`) or check the path of the file libcrypto.so.1.0.0 (`find / -name libcrypto.so.1.0.0`). If the file is not in `lib/x86...` then you should change the manifest (`python.manifest.template`).

And what do you mean by `SGX 2.1.3(1)` -- what are these numbers? The version of Intel SGX SDK? How did you obtain these numbers?

**Comment by gracequeen** [comment]
> Do you have the file `/lib/x86_64-linux-gnu/libcrypto.so.1.0.0` on your computer? Doesn't look like it. You probably should install it (`sudo apt-get install libssl1.0.0`) or check the path of the file libcrypto.so.1.0.0 (`find / -name libcrypto.so.1.0.0`). If the file is not in `lib/x86...` then you should change the manifest (`python.manifest.template`).
> 
> And what do you mean by `SGX 2.1.3(1)` -- what are these numbers? The version of` `Intel SGX SDK? How did you obtain these numbers?

**Comment by gracequeen** [comment]
Sorry about clicking the close button accidentally...
Thank you very much for reminding me to find `libcrypto.so.1.0.0` specifically! 
I've solved this problem. Here's what I did:
Search `libcrypto.so.1.0.0` in the system, and find it in `/usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.0`
Copy this one and `libssl.so.1.0.0` from `/usr/lib/x86_64-linux-gnu/` to `/lib/x86_64-linux-gnu/`

By `SGX 2.1.3(1)/(2)`, I was referring to the Readme.md in the root of your graphene repo... Sorry about the confusion.
